integration/spark/src/test/resources/carriagereturninstring.csv

```diff
@@ -0,0 +1,2 @@
+1,2
,3
+4,5,6
```

Comment:
suggest to name the csv file like 'carriage_return_in_string.csv' as the name so long without under score is confusing

Reply:
Done